packages/aws-cdk/test/integ/cli/bootstrapping.integtest.ts

```diff
@@ -1,52 +1,175 @@
-import { cloudFormation, deleteStacks } from './aws-helpers';
-import { cdk, cleanupOldStacks, fullStackName, prepareAppFixture } from './cdk-helpers';
+import { cloudFormation, deleteStacks, s3 } from './aws-helpers';
+import { cdk, cdkDeploy, cleanupOldStacks, deleteableStacks, fullStackName, prepareAppFixture } from './cdk-helpers';
 
-jest.setTimeout(600 * 1000);
+jest.setTimeout(600_000);
+
+// Needs to not start with the ACTUAL fullstack name otherwise cleanupOldStacks will pick it up.
+const BOOTSTRAP_STACK_PREFIX = `toolkit-${fullStackName('stack')}`;
+const BUCKETS_TO_DELETE = new Array<string>();
+const QUALIFIER = randomString();
```

Comment:
Can you rename these to regular camelCase? They're not really constants... (`BUCKETS_TO_DELETE` is mutated, `QUALIFIER` uses `randomString()`, etc.).